app/code/core/Mage/Adminhtml/Block/Sales/Order/Grid.php

```diff
@@ -141,21 +141,38 @@ protected function _prepareColumns()
             );
         }
 
+        $this->addRssFeedLink();
+
+        $this->addExportType('*/*/exportCsv', Mage::helper('sales')->__('CSV'));
+        $this->addExportType('*/*/exportExcel', Mage::helper('sales')->__('Excel XML'));
+
+        return parent::_prepareColumns();
+    }
+
+    /**
+     * Add link to RSS feed when enabled for filtered store-view
+     *
+     * @return $this
+     * @throws Mage_Core_Model_Store_Exception
+     */
+    public function addRssFeedLink()
+    {
         if (Mage::helper('sales')->isModuleOutputEnabled('Mage_Rss')) {
-            $filterString = $this->getParam($this->getVarNameFilter());
-            $filter = Mage::helper('adminhtml')->prepareFilterString($filterString);
-            $storeId = array_key_exists('store_id', $filter) ? $filter['store_id'] : null;
+            $storeId = null;
+
+            $filterString = $this->getParam($this->getVarNameFilter(), '');
+            if ($filterString) {
+                $filter = Mage::helper('adminhtml')->prepareFilterString($filterString);
+                $storeId = array_key_exists('store_id', $filter) ? $filter['store_id'] : null;
```

Comment:
I opt for readability and consistency:
```suggestion
                $storeId = $filter['store_id'] ?? null;
```
Is it better?

Reply:
Yes :)